src/Tgstation.Server.Host/Setup/SetupApplication.cs

```diff
@@ -52,7 +52,7 @@ public void ConfigureServices(IServiceCollection services)
 			if (services == null)
 				throw new ArgumentNullException(nameof(services));
 
-			services.SetupLogging(config => config.MinimumLevel.Override("Microsoft", LogEventLevel.Warning));
```

Comment:
This log setup is only for the setup wizard. I don't think it needs to be ingested into elastic. Also, making the `ElasticsearchConfiguration` available here would complicate things.